blockchain/chain.go

```diff
@@ -625,8 +625,9 @@ func (b *BlockChain) loadBlockNode(dbTx database.Tx, hash *chainhash.Hash) (*blo
 	}
 
 	blockHeader := block.MsgBlock().Header
-	node := newBlockNode(&blockHeader, ticketsSpentInBlock(block),
-		ticketsRevokedInBlock(block), voteBitsInBlock(block))
+	tickets := ticketsInBlock(block)
+	node := newBlockNode(&blockHeader, tickets.spentTickets, tickets.revokedTickets,
```

Comment:
i wonder newBlockNode should just take a ticketsInBlockInfo now...

Reply:
Could be helpful, yeah.

Do you wanna wait for someone else to chime in, or should I go ahead and do it?